lib/mlir/Dialect/Clift/IR/CliftOps.cpp

```diff
@@ -447,6 +447,14 @@ struct ModuleValidator {
                                     /*DirectlyNested=*/false))
         return Op->emitOpError()
                << Op->getName() << " must be nested within a loop operation.";
+    } else if (auto Sym = mlir::dyn_cast<MakeLabelOp>(Op)) {
+      if (not LocalSymbols.insert(Sym.getName()).second)
+        return Op->emitOpError()
+               << Op->getName() << " conflicts with another local symbol.";
+    } else if (auto Sym = mlir::dyn_cast<LocalVariableOp>(Op)) {
+      if (not LocalSymbols.insert(Sym.getSymName()).second)
+        return Op->emitOpError()
+               << Op->getName() << " conflicts with another local symbol.";
```

Comment:
I thought we agreed that local symbol name clashes should be checked separately for labels and local variables. Any reason why you're treating them together here?

Reply:
No, I just didn't implement this yet. Good thing you pointed it out here because I probably would have forgotten about it otherwise.

Reply:
This is now implemented.